doc/source/how-to-configure-run.rst

```diff
@@ -0,0 +1,127 @@
+############################
+ Configure a Flower App run
+############################
+
+**********************
+ Using the App config
+**********************
+
+A Flower App run can be configured using the ``tool.flwr.app.config``
+field of the ``pyproject.toml`` file.
+
+The syntax used is the standard TOML syntax:
+
+.. code:: toml
+
+   # Start of the `pyproject.toml` skipped
+
+   [tool.flwr.app.config]
+   num-server-rounds = 3
+   local-epochs = 1
+   lr = 0.1
+   verbose = true
+   run-name = "Small Run"
+
+Those values will then be accessible through the ``run_config``
+attribute of the ``Context`` object passed to the ``client_fn`` (for the
+``client_app``) and the ``server_fn`` (for the ``server_app``):
+
+.. code:: python
+
+   def client_fn(context: Context):
+       local_epochs = context.run_config["local-epochs"]
+       lr = context.run_config["lr"]
+       verbose = context.run_config["verbose"]
+
+       return CustomClient(local_epochs, lr, verbose)
+
+
+   app = ClientApp(client_fn=client_fn)
+
+.. code:: python
```

Comment:
```suggestion

Note that accessing the `run_config` to configure your `ServerApp` can be done in the same way as above:

.. code:: python
```

Reply:
I thought of adding some simple text so there is something between the two code cells. I think it looks better. maybe we want to shorten the sentence above the `client_fn()` example